### DISCUSSION TITLE
Dealing with arrays

### DISCUSSION THREAD

**Discussion by AirShoot125** [vfsfitvnm/frida-il2cpp-bridge#514] [Q&A] [answered]
in il2cpp game there's a function like this 
public static byte[] IKDCKFIPHDI(byte[] OGJFBCMENJH, byte[] JGHIEFIHNBJ, byte[] ALGAHFLNHMA) { }
I want to do something with these byte arrays that is taken by method as arguments, see/edit, how is it possible?

**Comment by thinhbuzz** [comment, marked as answer]
```ts
Il2Cpp.array(Il2Cpp.corlib.class('System.Byte'), [1,2,3,4]);
```
